src/AvtaleSide/GodkjenningSteg/Oppsummering/Avtaleparter/Avtaleparter.tsx

```diff
@@ -29,6 +27,26 @@ export const HarData = (input: string): React.ReactNode => {
     );
 };
 
+const sjekkNavnForConCatOmBeggeEksisterer = (
+    navn: string,
+    etternavn: string
+): string => {
+    if (sjekkStrengVerdi(navn) && sjekkStrengVerdi(etternavn)) {
```

Comment:
Holder det ikke med truthy-sjekk her, i stedet for funksjon?

Reply:
fikset